app/README.md

```diff
@@ -21,9 +21,17 @@
 3. **Install Playwright dependencies:**
    Run `bunx playwright install` to install Playwright dependencies.
 
+### Optional Setup
```

Comment:
@ehdlg Add an emoji (maybe the same as the main options works) for the title please.

Reply:
Done :+1: